modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter.po

```diff
@@ -0,0 +1,3 @@
+#: ../src/lightdm-gtk-greeter.c:1479
```

Comment:
I just realized this line will probably break if `lightdm-gtk-greeter` gets updated in the next Debian release.